src/applications/personalization/preferences/actions/index.js

```diff
@@ -82,8 +90,16 @@ export function setPreference(code, value = true) {
   };
 }
 
+export function setDismissedBenefitAlerts(value = []) {
```

Comment:
Dismissed alert information has been added to redux in order to update the view after removals on the dashboard, and updates on the checklist.